client/src/app/app.module.ts

```diff
@@ -41,6 +43,7 @@ import { LoginModule } from './modules/auth/auth.module';
     LoginModule,
     AppRoutingModule,
     CoreModule,
+    MomentModule
```

Comment:
@waaana not sure if this is the best place to declare this module after your refactoring (modularization)

Reply:
It is a good place, Point of the refactoring was to have AppComponent only declaration (and the components needed in the appcomponent). Modules are OK to be populated